EIPS/eip-4844.md

```diff
@@ -271,7 +271,7 @@ def point_evaluation_precompile(input: Bytes) -> Bytes:
     # Quotient kzg: next 48 bytes
     quotient_kzg = input[144:192]
     assert verify_kzg_proof(data_kzg, x, y, quotient_kzg)
-    return Bytes([])
+    return Bytes(U256(FIELD_ELEMENTS_PER_BLOB).to_bytes32() + U256(BLS_MODULUS).to_bytes32())
```

Comment:
Is this little or big endian? Should be clarified.

Reply:
Should be big endian as the EVM is?

Reply:
Yes, I will clarify in a quick separate PR